README.rst

```diff
@@ -2,7 +2,15 @@
 Stingray
 ========
 
-|Build Status Master| |Docs| |Slack| |joss| |doi| |Coverage Status Master| |GitHub release|
+.. csv-table::
+   :header: "Usage", "Release", "Development", "Community"
+   :widths: 50, 50, 50, 50
+
+   |Python version|, |GitHub release|, |Build Status Master|, |Slack|
+   |Docs|, |joss|, |Repo status|,  " "
+   |License|, |doi|, |Coverage Status Master|, " "
+
+
```

Comment:
@masonng-astro @taldcroft Here are the new badges, including the repo status from repostatus.org and the supported Python versions, as requested